### ISSUE TITLE
No menubar icons (macOS Monterey)

### ISSUE THREAD

**Issue by tylergannon** [exelban/stats#605]
**Describe the bug**

Nothing appears in the menubar.  I'm fully aware that I'm running a beta version of the operating system so no pressure on you, but thought you might like to be prepared for possible breaking changes in menubar functionality with Monterey.

**Details:**

<img width="596" alt="Screen Shot 2021-09-05 at 10 52 46 AM" src="https://user-images.githubusercontent.com/50593/132134950-fdb598c2-83ac-43f5-bd9f-823d92a18720.png">

 - Application version 2.6.8

**Comment by exelban** [comment]
Hi. Thanks for the feedback.
As you noticed, it's a beta version. I don't have a Monterey to check why the app does not work. 
But I still will wait for the release. (PS: I think it's more related to the Apple Silicon platform, not to the macOS).

Could you maybe check if you have a crash report of Stats? It could be found in `Applications -> Utilities -> Console`. Find the Crash Reports tab. There you must see a row called Stats.

**Comment by tylergannon** [comment]
Oh, that's an interesting hunch, that it's more related to the architecture change.
I don't see any crash reports in Console.app.  Here's a little more information though:

I started the app a few times, both by clicking on the icon in Finder, as well as using the `open -a /Applications/Stats.app/` from a terminal window.  The result was the same each time.  No error message, the settings window will appear, but nothing in the menu bar.

Now, here's the magic!

I rebooted my computer this morning, after logging in, there's the info in the menu bar...

![Screen Shot 2021-09-06 at 9 54 08 AM](https://user-images.githubusercontent.com/50593/132241644-ff3a6859-1823-422b-a6a0-6e6a44ecc8d5.png)

So maybe for the time being, it could be helpful to add advice into your README, that logging out / logging in can produce results for users who don't see changes immediately after installation.
